app/models/user.rb

```diff
@@ -116,4 +119,8 @@ def self.from_oauth_hash(auth_hash)
   def role_on(proposal)
     RolePicker.new(self, proposal)
   end
+
+  def requires_profile_attention?
```

Comment:
niiiiice. This might also work well on a `UserDecorator` if we have / want to use one of those, since it's being used purely for view logic